build/helm/keylime/charts/keylime-registrar/templates/_helpers.tpl

```diff
@@ -68,7 +68,7 @@ Expand to the name of the config map to be used
 {{- if .Values.global.configmap.create }}
 {{- include "keylime.configMap" . }}
 {{- else }}
-{{- default (include "keylime.configMap" .) .Values.global.configmap.registrarName }}meah
+{{- default (include "keylime.configMap" .) .Values.global.configmap.registrarName }}
```

Comment:
meah :) ... love it

Reply:
I thought when removing this it has no critical impact.